index.html

```diff
@@ -10,219 +10,268 @@
-            <h3 class="project-title">Multi-Post Stories</h3>
-            <ul class="canopy">
-            <li>CANOPY</li>
-            <li class="bullets"><span class="dot"></span></li>
-            <li class="o-li">Back End Dev</li>
-            <li class="bullets"><span class="dot"></span></li>
-            <li class="o-li">2015</li>
-            </ul>
-            </div>
-            <p>
-            A daily selection of privately personalized reads; no accounts or
-            sign-ups required.
-            </p>
-            <ul class="categories hex">
-            <li>html</li>
-            <li>css</li>
-            <li>javaScript</li>
-            </ul>
             <div class="see-project">
-            <button type="button" class="hex">See Project</button>
-            </div>
-            </div>
-            </section>
-            <section class="grid-items other-projects">
-            <img class="project-img" src="images/projects/project-3.png" alt="Project-1" />
-            <div class="details">
-            <div class="titles">
-            <h3 class="project-title">Tonic</h3>
-            <ul class="canopy">
-            <li>CANOPY</li>
-            <li class="bullets"><span class="dot"></span></li>
-            <li class="o-li">Back End Dev</li>
-            <li class="bullets"><span class="dot"></span></li>
-            <li class="o-li">2015</li>
-            </ul>
-            </div>
-            <p>
-            A daily selection of privately personalized reads; no accounts or
-            sign-ups required.
-            </p>
-            <ul class="categories hex">
-            <li>html</li>
-            <li>css</li>
-            <li>javaScript</li>
-            </ul>
-            <div class="see-project">
-           <button type="button" class="hex">See Project</button>
-            </div>
-            </div>
-            </section>
-            <section class="grid-items other-projects">
-            <img class="project-img" src="images/projects/project-4.png" alt="Project-1" />
-            <div class="details">
-            <div class="titles">
-            <h3 class="project-title">Multi-Post Stories</h3>
-            <ul class="canopy">
-            <li>CANOPY</li>
-            <li class="bullets"><span class="dot"></span></li>
-            <li class="o-li">Back End Dev</li>
-            <li class="bullets"><span class="dot"></span></li>
-            <li class="o-li">2015</li>
-            </ul>
-            </div>
-            <p>
-            A daily selection of privately personalized reads; no accounts or
-            sign-ups required.
-            </p>
-            <ul class="categories hex">
-            <li>html</li>
-            <li>css</li>
-            <li>javaScript</li>
-            </ul>
-            <div class="see-project">
-            <button type="button" class="hex">See Project</button>
+                <button type="button" class="hex">Get my resume</button>
             </div>
+
```

Comment:
- [ ] [OPTIONAL] Kindly fix you design width as it is over flowing, either by making the body's width 100vw or some other better way you can get by making your research 😉.